mega-zep-backend/src/main/java/com/gepardec/mega/zep/ZepServiceImpl.java

```diff
@@ -99,6 +103,44 @@ public void updateEmployeesReleaseDate(final String userId, final String release
         return projectTimeMapper.mapToEntryList(projectTimeResponse.getProjektzeitListe().getProjektzeiten());
     }
 
+    @Override
+    public List<Project> getProjectsForYear(final LocalDate monthYear) {
+        final ReadProjekteResponseType readProjekteResponseType = getProjectsInternal(monthYear);
+        final List<Project> projects = new ArrayList<>();
+
+        readProjekteResponseType.getProjektListe().getProjekt()
```

Comment:
Maybe split to multiple methods for better structuring and readability

Reply:
refactored